scripts/deployment/README.md

```diff
@@ -102,49 +66,78 @@ uses the subnet and all VM instances in MIG run on private IP only.
- creates the MIG and deploys SDRS into the MIG.   
-    
-         Follow the steps section I (subsection 1 through 11) to create a Managed Instance Group.    
-    
-4. Deploying the [Endpoints configuration](#cloud-endpoints) using the following command.    
+The following are instructions to deploy SDRS. Click the links for details on each step:     
+
+1. Create a [CloudSQL](./cloud-sql/README.md) instance. (Takes about 10-15 minutes).
+2. Set [log_bin_trust_function_creators](https://stackoverflow.com/questions/47359508/cant-create-trigger-on-mysql-table-within-google-cloud) to true.
+
+        gcloud sql instances patch <CLOUDSQL_INSTANCE_NAME> --database-flags log_bin_trust_function_creators=on
+
+3. Run [SQL DDL](../sql/retention_schema.sql) to create a database schema in Cloud SQL created above.
+
+    > Connecting to Cloud SQL over Private IP will require a bastion host within the VPC or a private connection like VPN from origin to the VPC
+
+    > Once the Cloud SQL is reachable, the below command can be run to create the database schema:
+
+        mysql --host=<CLOUDSQL_PRIVATE_IP> --user=root --password < retention_sdrs.sql
+
+
+4. Create a [Pub/Sub topic](./pub-sub) for SDRS to publish messages. 
+5. Create a [custom image](#custom-image) to be used by MIG.  
+6. Create a [Docker Container Image](#docker-container-image) containing SDRS application build.
+7. Launch a [Managed Instance Group cluster](./mig/README.md) (Takes about 5 minutes ). This step creates the MIG and deploys SDRS into the MIG.   
     
-          gcloud endpoints services deploy openapi.yaml    
-          
+        Follow the steps section I (subsection 1 through 11) to create a Managed Instance Group.    
+   
+8. Deploying the [Endpoints configuration](#cloud-endpoints)
+
+9. Verify the status of SDRS using [Check SDRS Status](#check-sdrs-status)
+
 ### SDRS Update    
- Update the SDRS application using [MIG Updater](./mig/README.md).    
+Update the SDRS application using [MIG Updater](./mig/README.md).    
  
- Follow the steps section II (subsection 1 through 4) to update the version of new software on the existing Managed Instance Group.    
-    
+Follow the steps section II (subsection 1 through 4) to update the version of new software on the existing Managed Instance Group.    
  
 > Notes:
->  1. The Managed Instance Group updater does NOT cause a change of Internal Load Balancer Private IP Address or Endpoints configuration
+> 1. The Managed Instance Group updater does NOT cause a change of Internal Load Balancer Private IP Address or Endpoints configuration
 > change.    
 > 2. The Managed Instance Group Infrastructure is immutable i.e. when you update the MIG with a newer software version the VM instances are
 > replaced by the new VM instances running newer software version.    
 > 3. You can not stop or restart VM instances in a MIG.
 
+### SDRS Monitoring
```

Comment:
this section is more about JVM Monitoring. can we change the section title to SDRS JVM Monitoring